CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.5.0
+
+- [#32](https://github.com/patterninc/muffin_man/pull/32)
+- Removes `keywords` variable requirement for v20220401 search_catalog_items as it is not required in v20220401. You'll need to pass `keywords` as a param rather than as an argument for the method
```

Comment:
Maybe to make it a little more obvious when we release a version with breaking changes we call them out with something like:

```suggestion
## Breaking changes - 1. v20220401 search_catalog_items method signature updated

- [#32](https://github.com/patterninc/muffin_man/pull/32) Removes `keywords` argument requirement for v20220401 search_catalog_items as it is not required in v20220401. You'll need to pass `keywords` as a param rather than as an argument for the method
```